src/cluster/src/install.rs

```diff
@@ -100,7 +101,7 @@ impl ClusterInstallerBuilder {
 
     /// Sets the Kubernetes namespace to install Fluvio into.
     ///
-    /// The default namespace is "default".
+    /// The default namespace is "fluvio".
```

Comment:
should not use fluvio namespace as default since we can't assume user wants to use fluvio.  it should be opt-in